src/System.Device.Gpio.Tests/GpioControllerTestBase.cs

```diff
@@ -95,7 +95,22 @@ public void IsPinOpenTest()
                 controller.ClosePin(LedPin);
                 Assert.False(controller.IsPinOpen(LedPin));
             }
-        }
+			using (GpioController controller = new GpioController(GetTestNumberingScheme(), GetTestDriver()))
+			{
+				Assert.False(controller.IsPinOpen(LedPin));
+
+				controller.OpenPin(LedPin, PinMode.Output);
+				Assert.True(controller.IsPinOpen(LedPin));
```

Comment:
Also please make sure that you use spaces instead of tabs.

Reply:
Yea, sorry. My VS is set to tabs by default, but for some reason the .editorConfig setting is ignored.